src/wp-includes/class-wp-comment-query.php

```diff
@@ -770,6 +771,15 @@ protected function get_comment_ids() {
 			'NOT IN' => (array) $this->query_vars['type__not_in'],
 		);
 
+		// @since 6.9.0 Exclude the 'note' comment type, unless 'all' types or the 'note' type explicitly are requested.
```

Comment:
Since this isn't a phpdoc comment, maybe the `@since` part should be removed?

```suggestion
		// Exclude the 'note' comment type, unless 'all' types or the 'note' type explicitly are requested.
```